src/modules/mavlink/mavlink_receiver.h

```diff
@@ -184,94 +191,83 @@ class MavlinkReceiver : public ModuleParams
+	uORB::Publication<battery_status_s>			_battery_pub{ORB_ID(battery_status)};
+	uORB::Publication<collision_report_s>			_collision_report_pub{ORB_ID(collision_report)};
+	uORB::Publication<debug_array_s>			_debug_array_pub{ORB_ID(debug_array)};
+	uORB::Publication<debug_key_value_s>			_debug_key_value_pub{ORB_ID(debug_key_value)};
+	uORB::Publication<debug_value_s>			_debug_value_pub{ORB_ID(debug_value)};
+	uORB::Publication<debug_vect_s>				_debug_vect_pub{ORB_ID(debug_vect)};
+	uORB::Publication<follow_target_s>			_follow_target_pub{ORB_ID(follow_target)};
+	uORB::Publication<landing_target_pose_s>		_landing_target_pose_pub{ORB_ID(landing_target_pose)};
+	uORB::Publication<obstacle_distance_s>			_obstacle_distance_pub{ORB_ID(obstacle_distance)};
+	uORB::Publication<offboard_control_mode_s>		_offboard_control_mode_pub{ORB_ID(offboard_control_mode)};
+	uORB::Publication<optical_flow_s>			_flow_pub{ORB_ID(optical_flow)};
+	uORB::Publication<position_setpoint_triplet_s>		_pos_sp_triplet_pub{ORB_ID(position_setpoint_triplet)};
+	uORB::Publication<vehicle_attitude_s>			_attitude_pub{ORB_ID(vehicle_attitude)};
+	uORB::Publication<vehicle_attitude_setpoint_s>		_att_sp_pub{ORB_ID(vehicle_attitude_setpoint)};
+	uORB::Publication<vehicle_global_position_s>		_global_pos_pub{ORB_ID(vehicle_global_position)};
+	uORB::Publication<vehicle_gps_position_s>		_gps_pub{ORB_ID(vehicle_gps_position)};
+	uORB::Publication<vehicle_land_detected_s>		_land_detector_pub{ORB_ID(vehicle_land_detected)};
+	uORB::Publication<vehicle_local_position_s>		_local_pos_pub{ORB_ID(vehicle_local_position)};
+	uORB::Publication<vehicle_odometry_s>			_mocap_odometry_pub{ORB_ID(vehicle_mocap_odometry)};
+	uORB::Publication<vehicle_odometry_s>			_visual_odometry_pub{ORB_ID(vehicle_visual_odometry)};
+	uORB::Publication<vehicle_rates_setpoint_s>		_rates_sp_pub{ORB_ID(vehicle_rates_setpoint)};
+	uORB::Publication<vehicle_trajectory_waypoint_s>	_trajectory_waypoint_pub{ORB_ID(vehicle_trajectory_waypoint)};
+
+	// ORB publications (multi)
+	uORB::Publication<distance_sensor_s>			_distance_sensor_pub{ORB_ID(distance_sensor), ORB_PRIO_LOW};
+	uORB::Publication<distance_sensor_s>			_flow_distance_sensor_pub{ORB_ID(distance_sensor), ORB_PRIO_LOW};
+	uORB::Publication<input_rc_s>				_rc_pub{ORB_ID(input_rc), ORB_PRIO_LOW};
+	uORB::Publication<manual_control_setpoint_s>		_manual_pub{ORB_ID(manual_control_setpoint), ORB_PRIO_LOW};
+	uORB::Publication<ping_s>				_ping_pub{ORB_ID(ping), ORB_PRIO_LOW};
+	uORB::Publication<radio_status_s>			_radio_status_pub{ORB_ID(radio_status), ORB_PRIO_LOW};
+	uORB::Publication<sensor_accel_s>			_accel_pub{ORB_ID(sensor_accel), ORB_PRIO_LOW};
+	uORB::Publication<sensor_baro_s>			_baro_pub{ORB_ID(sensor_baro), ORB_PRIO_LOW};
+	uORB::Publication<sensor_gyro_s>			_gyro_pub{ORB_ID(sensor_gyro), ORB_PRIO_LOW};
+	uORB::Publication<sensor_mag_s>				_mag_pub{ORB_ID(sensor_mag), ORB_PRIO_LOW};
+
+	// ORB publications (queue length > 1)
```

Comment:
Nice work on the organization/alphabetization!  It's really easy to spot potential duplicates and inspect them.  Super nice to read through. Thanks!